Automattic/woocommerce-payments
# Payouts: improve spotlight notice illustration (use the new design specific to payouts rename)

Fixes #9700 

#### Changes proposed in this Pull Request

Change the illustration on the spotlight notice announcing renaming `deposit` to `payout`, as per Figma design.

#### Testing instructions

* Check in to this branch and rebuild assets.
* The spotlight notice should look like the screenshot below. 

<img src="https://github.com/user-attachments/assets/3b4d4716-eff2-4719-b755-371cd755c0f6" width="500px" /> 

-------------------


**Post merge**
